packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/studio/services/library/StudioLibraryPublicationHandler.java

```diff
@@ -69,14 +72,17 @@ public class StudioLibraryPublicationHandler implements ILibraryPublicationHandl
 
     private final IProjectSemanticDataSearchService projectSemanticDataSearchService;
 
+    private final ILibrarySearchService librarySearchService;
+
     private final IMessageService messageService;
 
-    public StudioLibraryPublicationHandler(IEditingContextSearchService editingContextSearchService, ISemanticDataCreationService semanticDataCreationService, IResourceToDocumentService resourceToDocumentService, IStudioLibraryDependencyCollector studioLibraryDependencyCollector, IProjectSemanticDataSearchService projectSemanticDataSearchService, IMessageService messageService) {
+    public StudioLibraryPublicationHandler(IEditingContextSearchService editingContextSearchService, ISemanticDataCreationService semanticDataCreationService, IResourceToDocumentService resourceToDocumentService, IStudioLibraryDependencyCollector studioLibraryDependencyCollector, IProjectSemanticDataSearchService projectSemanticDataSearchService, ILibrarySearchService librarySearchService, IMessageService messageService) {
```

Comment:
I updated this class to take into account imported libraries in the publication process (we don't want to publish them, we want to create dependencies to them)